developers/weaviate/client-libraries/typescript/v2_v3_migration.md

```diff
@@ -83,15 +55,15 @@ To go from `v2` to `v3`, you must
 
 ## Instantiation
 
-The `v3` client is instantiated through the `WeaviateClient` object, which is the main entry point for all API operations.
+The `v3` client is instantiated through the `weaviate` object, which is the main entry point for all API operations.
```

Comment:
Consider:

The `weaviate` object is the main entry point for all API operations. The `v3` client instantiates the `weaviate` object.